config/groups.yaml

```
@@ -120,6 +120,12 @@
     - "2666525"
     - "2328859"
   id: 189749
+
+  -name: "SSH IDV"
+   members:
+     - "2601703"
+     - "2606037"
+     - "2606308"
```

You have too much indentation:
```suggestion
- name: "SSH IDV"
  members:
    - "2601703"
    - "2606037"
    - "2606308"
```